### PR TITLE
a2dismod, a2enmod: add pt_PT translation

### PR DESCRIPTION
**Pull request by tcarrondo**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_PT/linux/a2dismod.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# a2dismod
+
+> Desactiva um módulo do Apache nas distribuições baseadas em Debian.
+> Mais informações: <https://manpages.debian.org/latest/apache2/a2dismod.8.en.html>.
+
+- Desactiva um módulo:
```
[the remark this answers is not in the document]
Comment (waldyrious) [reply pages.pt_PT/linux/a2dismod.md:6]: Update: as per the discussion in #7159, the mood change is not required at the moment.

@tcarrondo any thoughts about the orthography change?

**Comment by tcarrondo** [comment]
I'm a convict non-user of the new orthography.
In all translations I do, I don't use new orthography because it's very easy to change to new orthography with a simple sed, but the opposite isn't that simple. I would like to see applied here the solution adopted by projects like WordPress, that split translations in Portuguese and Portuguese (AO1990) the last one done with a script.

**Comment by waldyrious** [comment]
I don't think there are strong opinions among the Portuguese contributors and maintainers regarding the AO, so I'm happy to heed your preference (though I wouldn't at this point impose the pre-AO spelling on other contributions).

> I would like to see applied here the solution adopted by projects like WordPress, that split translations in Portuguese and Portuguese (AO1990) the last one done with a script.

Honestly, given (1) the relatively small amount of Portuguese pages, (2) the already existing division between pt-PT and pt-BR (which splits the contributor pool), and (3) the language-specific nature of this distinction (which would open a precedent to attempt to model the complexities for other languages), I would recommend against keeping both pre-AO and post-AO versions of the pages explicitly in the repo.

That said, it seems like a good feature for a tldr-pages client to support a configuration (or CLI option) to toggle the automated post-AO conversion that you mention as a runtime step.

File: pages.pt_PT/linux/a2dismod.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (waldyrious) [reply pages.pt_PT/linux/a2dismod.md:6]: Per the discussion [below](https://github.com/tldr-pages/tldr/pull/7177#issuecomment-962527055), I'm dropping the suggestion to use the reformed orthography.

File: pages.pt_PT/linux/a2dismod.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# a2dismod
+
+> Desactiva um módulo do Apache nas distribuições baseadas em Debian.
```
[the remark this answers is not in the document]
Comment (waldyrious) [reply pages.pt_PT/linux/a2dismod.md:3]: @tcarrondo this is a very minor change and we could merge without it, but I'd rather have your opinion.

**Comment by tcarrondo** [comment]
@waldyrious all done. thanks for all your help.

**Review by waldyrious — APPROVED** [review]
Let's go! Thanks again for the contribution, and for the patience during the review :)

**Pull request merged**